src/main/java/com/wenqi/bootwebdemo/dao/PersonRepo.java

```diff
@@ -0,0 +1,174 @@
+package com.wenqi.bootwebdemo.dao;
+
+import com.wenqi.bootwebdemo.model.Person;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class PersonRepo {
+    private static final Logger logger = LogManager.getLogger(PersonRepo.class);
+    // JDBC driver name and database URL
+    @Value("${JDBC_DRIVER}")
+    String JDBC_DRIVER;
```

Comment:
reserve all caps naming style for constants, here can just be "jdbcDriver"

Reply:
Will change person repo methods to access db (change to JDBC template usage)